src/components/Comments.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { fetchComments } from "./Api";
+
+
+const Comments = ({article_id}) => {
+    const [comments, setComments] = useState([])
+    const [isLoading, setIsLoading] = useState(true);
+    useEffect(() => {
+        fetchComments(article_id).then((data) => {
+            setComments(data.comments)
+            setIsLoading(false)
+        })
+    })
```

Comment:
Missing your dependency array - this is constantly spamming your api at the moment.